### ISSUE TITLE
Python: Ability to submit punctuation in streamsx.spl.spl.PrimitiveOperator

### ISSUE THREAD

**Issue by markheger** [IBMStreams/streamsx.topology#2518]
Provice method `submit_punct` for SPL Python Primitive Operator to emit window punctuation markers.

Task for #1982

```
@spl.primitive_operator(output_ports=['A'])
class SingleOutputPortForwardTupleAndSendPunct(spl.PrimitiveOperator):
    def __init__(self):
        pass

    @spl.input_port()
    def port0(self, *t):
        self.submit('A', t)
        self.submit_punct('A')
```

**Comment by markheger** [comment]
merged to branch feature/punct

**Comment by markheger** [comment]
merged to develop branch for 1.16.0a
